addons/resource_mail/static/tests/resource_mail_test_helpers.js

```diff
@@ -0,0 +1,13 @@
+import { ResourceTask } from "./mock_server/mock_models/resource_task";
+import { ResourceResource } from "./mock_server/mock_models/resource_resource";
```

Comment:
Technically these models should be in addon `resource/` rather than `resource_mail/`.
But honestly not a huge deal